info/v2/container.go

```diff
@@ -86,8 +86,11 @@ type ContainerStats struct {
 	// The time of this stat point.
 	Timestamp time.Time `json:"timestamp"`
 	// CPU statistics
-	HasCpu bool        `json:"has_cpu"`
-	Cpu    v1.CpuStats `json:"cpu,omitempty"`
+	HasCpu bool `json:"has_cpu"`
+	// In nanoseconds (aggregated)
+	Cpu v1.CpuStats `json:"cpu,omitempty"`
+	// In nanocores per second (instantaneous)
+	CpuInst v1.CpuStats `json:"cpu_inst,omitempty"`
```

Comment:
I think we shouldn't overload CpuStats to work for both. Dropping units in the base definition is confusing.
Let's split them and we can evolve them separately. 

Overall supporting instant metrics in cAdvisor SG.